.bazelrc

```diff
@@ -19,6 +19,7 @@ build --action_env=BAZEL_LINKLIBS=-l%:libstdc++.a
 build --action_env=BAZEL_LINKOPTS=-lm
 build --host_javabase=@bazel_tools//tools/jdk:remote_jdk11
 build --javabase=@bazel_tools//tools/jdk:remote_jdk11
+build --copt=-fPIC
```

Comment:
Is it possible to add a Python or shell test that verifies the PIE property on either the binary produced by Bazel or when actually running? Given prior discussions, belts-and-braces is ideal if we can do it reasonably.